packages/api/src/command/medical/mapi-access.ts

```diff
@@ -8,6 +8,12 @@ export async function allowMapiAccess(cxId: string): Promise<"new" | "existing">
   return "new";
 }
 
+export async function getMapiAccess(cxId: string): Promise<boolean> {
```

Comment:
since the return is a boolean, maybe the name of the func should be something like `isMapiAccessEnabled` or `doesCxHaveMapiAccess`? 

`getMapiAccess` sounds like it will provide mapi access for a cx that doesn't yet have it

Reply:
Good point. Fixed!